openwisp_firmware_upgrader/tasks.py

```diff
@@ -15,6 +16,60 @@
 logger = logging.getLogger(__name__)
 
 
+@shared_task(base=OpenwispCeleryTask, bind=True)
+def delete_firmware_files(self, file_paths):
+    """
+    Deletes firmware files from storage in the background
+    """
+    logger.info(f"Starting delete_firmware_files task with paths: {file_paths}")
```

Comment:
I think the logging here is too verbose. Either remove the obvious log lines or change the log level to DEBUG as it will create unnecessary noise.